### PR TITLE
Hera+

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: libevm/VMFactory.cpp [hunk 1]
```diff
@@ -54,6 +55,7 @@ VMKindTableEntry vmKindsTable[] = {
 #endif
 #if ETH_HERA
     {VMKind::Hera, "hera"},
+    {VMKind::HeraPlus, "hera+"},
```
Comment (axic) [inline libevm/VMFactory.cpp]: Should this be `heraplus` on the command instead?

**Comment by axic** [comment]
Probably the most generic version would be "ChainedVM" which has a list of `VMKind`s which are executed in order (until one responds with "non-UNSUPPORTED-BYTCODE")

**Comment by axic** [comment]
@chfast you didnt update the evmjit repo to the right version, it doesn't have the kind value

File: libevm/VMFactory.cpp (hunk 1 above)
Comment (chfast) [reply libevm/VMFactory.cpp]: Can be  :)

**Pull request merged**